app/models/participacao.rb

```diff
@@ -1,6 +1,7 @@
 class Participacao < ActiveRecord::Base
   belongs_to :usuario
   belongs_to :acao
+  delegate :nome, :to => :acao, :allow_nil => false
```

Comment:
Use o prefix => true, e allow_nil => true. 
- O allow_nil true é importante, porque ele irá tratar caso o objeto esteja nulo ele retorna nil ao invés de um exception. 
- O prefix é para você usar o objeto dessa forma no seu if participacao.acao_nome. 
Dessa forma respeitamos a lei de Demeter 😉

Segue um link muito bacana sobre a Lei de Demeter, depois de uma lida é bem bacana.
[](http://www.virtuouscode.com/2011/07/05/demeter-its-not-just-a-good-idea-its-the-law/)